lua/autorun/server/sv_cfc_notifications.lua

```diff
@@ -0,0 +1,57 @@
+util.AddNetworkString( "CFC_ReceivePopupNotification" )
+
+CFCNotifications = CFCNotifications or {}
+
+CFCNotifications._timerNameCount = 1
+
+local function getTimerName()
+    local newName = "notification-timer-" .. tostring( CFCNotifications._timerNameCount )
+
+    CFCNotifications._timerNameCount = CFCNotifications._timerNameCount + 1
+
+    return newName
+end
+
+local function makePopup( message, filter )
+    local getPlayers = filter or player.GetAll
+
+    local players = getPlayers()
+
+    net.Start( "CFC_PopupNotification" )
+        net.WriteString( message )
+    net.Send( players )
+end
+
+CFCNotifications._registerNotification = function( notificationType, message, delay, recurring, filter )
+    local repititions = recurring and 0 or 1
+
+    local callback = nil
+    if notificationType == "popup" then
```

Comment:
```suggestion
    if notificationType == "popup" then callback = makePopup end
```

Reply:
can you make this a multiline comment so I can commit it